<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1520</width>
    <height>890</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QVBoxLayout" stretch="0,0,0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="labelDrumMachine">
        <property name="text">
         <string>&lt;b&gt;DrumMachine&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSettings">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>⚙️</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelMaster">
        <property name="text">
         <string>Master:</string>
        </property>
        <property name="buddy">
         <cstring>horizontalSliderMaster</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSliderMaster">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxAudioDevice"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRefreshAudioDevices">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>↻</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxBufferSize">
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAudioDevice">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelMidiIn">
        <property name="text">
         <string>Midi in:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxMidiIn"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRefreshMidiIn">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>↻</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonMidiIn">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelMidiOut">
        <property name="text">
         <string>Midi out:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxMidiOut"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRefreshMidiOut">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>↻</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonMidiOut">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <widget class="DrumPadWidget" name="drumPadWidget">
       <attribute name="title">
        <string>DrumPad</string>
       </attribute>
      </widget>
      <widget class="LoopStationWidget" name="loopStationWidget">
       <attribute name="title">
        <string>LoopStation</string>
       </attribute>
      </widget>
      <widget class="DjWidget" name="djWidget">
       <attribute name="title">
        <string>DJ</string>
       </attribute>
      </widget>
      <widget class="SynthisizerWidget" name="synthisizerWidget">
       <attribute name="title">
        <string>Synthisizer</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1520</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DjWidget</class>
   <extends>QWidget</extends>
   <header>widgets/djwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DrumPadWidget</class>
   <extends>QWidget</extends>
   <header>widgets/drumpadwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SynthisizerWidget</class>
   <extends>QWidget</extends>
   <header>widgets/synthisizerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LoopStationWidget</class>
   <extends>QWidget</extends>
   <header>widgets/loopstationwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
